boilerplates/react-union-boilerplate-basic/src/widgets/Content/route.js

```diff
@@ -2,7 +2,7 @@ import loadWidget from './content.widget';
 
 export default {
 	path: 'content',
-	getComponents: cb => {
-		loadWidget(module => cb(module.default));
+	getComponent: done => {
+		loadWidget(mod => done(mod.default));
```

Comment:
use `module` instead of `mod`

Reply:
I found `module` quite distracting because of syntax highlighting, to be honest.